### PR TITLE
T05- Prime number checker

### PR DESCRIPTION
**Pull request by Priyanka15-12**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter a number: ");
+         int n = int.Parse (Console.ReadLine ());
+         int i = 1, count = 1;
+         while (n >= i) {
```
Comment (MetaNaveen) [inline Program.cs:7]: The iteration count could be reduced further.

File: Program.cs [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+            i++;
```
Comment (MetaNaveen) [inline Program.cs:8]: 'i' increment could be done in while loop condition.

File: Program.cs [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+            if (n % i == 0) {
```
Comment (MetaNaveen) [inline Program.cs:9]: Curly braces note required for single line statement.

**Review by MetaNaveen — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 1 above)
Comment (Priyanka15-12) [reply Program.cs]: Iteration reduced

File: Program.cs (hunk 2 above)
Comment (Priyanka15-12) [reply Program.cs]: Done

**Review by MetaNaveen — APPROVED** [review]

**Comment by MetaNaveen** [comment]
Working as expected.

**Final Notes:**
The condition "if (n % 2 != 0 && n > 2)" could be removed as the condition was satisfied by the previous conditions.

**Pull request closed without merging**